odkx-src/sync-endpoint.rst

```diff
@@ -89,12 +89,13 @@ ODK-X Sync Endpoint does not store user information in its own database, instead
 
 HTTPS
 -----
+  HTTPS stands for Hyper Text Transfer Protocol Secure. It is a protocol for securing the communication between two systems e.g. the browser and the web server. To know more about HTTPS and its working have a look at this video `Working of HTTPS <https://www.youtube.com/watch?v=T4Df5_cojAs>`_.
```

Comment:
@ShristiJain can you confirm this HTTPS section is the only one with content changes? It looks like you cleaned up some extra spaces elsewhere, just wanted to check if I missed any other content changes.

Reply:
Minor edits: "To learn more about HTTPS and how it works see this video" rather than "To know more about HTTPS and its working have a look at this video"